src_web/kamaho-shokusu/tests/TestCase/Service/NotificationServiceTest.php

```
@@ -0,0 +1,99 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Test\TestCase\Service;
+
+use App\Service\NotificationService;
+use Cake\TestSuite\TestCase;
+
+class NotificationServiceTest extends TestCase
```

PHPStan (2.2.2)

[error] Class App\Test\TestCase\Service\NotificationServiceTest extends unknown class Cake\TestSuite\TestCase.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)